pkg/k8s/watchers/node.go

```diff
@@ -37,11 +38,13 @@ func (k *K8sWatcher) nodesInit(k8sClient kubernetes.Interface) {
 		0,
 		cache.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
+				var valid bool
 				if node := k8s.ObjToV1Node(obj); node != nil {
-					if option.Config.BGPAnnounceLBIP {
-						k.bgpSpeakerManager.OnUpdateNode(node) // Need to seed the BGP speaker
-					}
```

Comment:
Why was this removed? It is still needed to seed the BGP subsystem. You can add it to the end of `updateK8sNodeV1`

Reply:
It's already called in `updateK8sNodeV1`. Is something different needed?

Reply:
Oh, right! I missed that. Nevermind :sweat_smile: